chall08/spentti.c

```diff
@@ -0,0 +1,15 @@
+int ft_ie_except_after_c(char *str)
+{
+	int i;
+
+	i = 0;
+	while (str[i + 1] != '\0')
+	{
+		if (str[i] == 'c' && str[i + 1] == 'i' && str[i + 2] == 'e')
+			return (0);
+		if ((i == 0 || str[i - 1] != 'c') && str[i] == 'e' && str[i + 1] == 'i')
+			return (0);
```

Comment:
Seems to have all necessary checks and nothing else, very good. One small optimization might be to check str[i] == 'e' first since (i == 0 || str[i - 1] != 'c') is true in a lot more cases so function makes more checks. Really small difference, but everything else is optimized nicely so thought I would point this out even if there's no difference for end result :)